mitmproxy/utils/debug.py

```diff
@@ -5,18 +5,24 @@
 import signal
 import platform
 import traceback
+import subprocess
 
 from mitmproxy import version
 
 from OpenSSL import SSL
 
 
-def sysinfo():
+def dump_system_info():
+    old_dir = os.getcwd()
+    os.chdir(os.path.abspath(os.path.join(os.path.dirname(__file__), "..")))
```

Comment:
Where would you put that function if we want to move it from `test/` into `mitmproxy/`?